portable_concurrency/bits/then.h

```diff
@@ -214,7 +227,7 @@ class cnt_state final:
 private:
   either<cnt_closure<F, T>, stored_cnt_res> storage_;
   std::exception_ptr exception_ = nullptr;
-  continuations_stack continuations_;
+  continuations_stack<std::allocator<void>> continuations_;
```

Comment:
http://en.cppreference.com/w/cpp/memory/allocator `std::allocator<void>` specialization is marked as deprecated in C++17. I think it would be better to use `std::allocator<typename future_state<T>::continuation>` here. Maybe with some alias to improve code readability.

Reply:
Same for: https://github.com/VestniK/portable_concurrency/pull/7/files#diff-278a734a5d98dd906ccb561cdc7e4b0cR91
And you can use `std::allocator<T>` in https://github.com/VestniK/portable_concurrency/pull/7/files#diff-871bb94f013758c50851ec24e7f7b99fR23